src/content/interop/js-interop/usage.md

```diff
@@ -389,6 +389,47 @@ but can often be elided on and within interop types and on extension members as
 the compiler can tell it is a JS interop type from the representation type and
 on-type.
 
+## Exporting Dart code to JS
+
+The above sections show how to call JS members from Dart. It's also useful to
+*export* Dart code so that it can be used in JS. The mechanism to do this
+involves exporting Dart functions, which requires a conversion like
+[`Function.toJS`] that wraps the Dart function with a JS function. Once
+converted, we can pass the resulting JS function to JS through an interop
+member, which can then be called later in JS.
```

Comment:
```suggestion
To export a Dart function to JS, first convert it using [`Function.toJS'], which wraps the Dart function with a JS function. Then, pass the wrapped function to JS through an interop member. At that point, it's ready to be called by other JS code.
```

Reply:
Done, but I like keeping the first two sentences here to contrast what this section is doing compared to the rest of interop.